### PR TITLE
Use angular-chosen instead of ui-select for better performance.

### LINKED ISSUE #2006: Fix empty users dropdown fields
To reproduce:
* import >50 users
* create a new motion and select the last user as submitter (is not in scoll list becase only the first 50 users are displayed)
* open motion update form 
-> submitter field shows empty user (because this user is filtered by 'limitTo')

### PR DESCRIPTION
**Pull request by emanuelschuetze**
The jQuery select field from angular-chosen is used for all ui-select
fields. See https://github.com/leocaseiro/angular-chosen/

Use input delay on users filter field with 'debounce'.

Fixed #2006.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -68,6 +68,12 @@ gulp.task('fonts-libs', function() {
         .pipe(gulp.dest(path.join(output_directory, 'fonts')));
 });
 
+// Catches img file for angular-chosen.
```
Comment (normanjaeckel) [inline gulpfile.js]: Is there more than one file? If yes, type `// Catches image files for angular-chosen.`

**Comment by normanjaeckel** [comment]
One comment, rest looks good.

File: gulpfile.js (hunk 1 above)
Comment (emanuelschuetze) [inline gulpfile.js]: Done.

**Comment by emanuelschuetze** [comment]
All done. I added a new commit which improved some translation strings. Please review 2nd commit. If ok I'll rebase and merge.

**Comment by normanjaeckel** [comment]
Ok. Rebase and merge.

**Pull request merged**